src/ide-sidecar.d.ts

```diff
@@ -1,7 +1,5 @@
 declare module "ide-sidecar" {
-  /** Path to ide-sidecar binary file. The version is resolved at build time. */
-  const path: string;
   /** The ide-sidecar version is resolved at build time. */
   export const version: string;
-  export default path;
+  export default version;
 }
```

Comment:
Seems like we could just set the version as an environment variable like we do for `SENTRY_ENV` rather than keep this around